vertica_python/vertica/connection.py

```diff
@@ -161,7 +165,7 @@ def peek(self):
                 # add resolved IP addresses to deque
                 for res in reversed(resolved_hosts):
                     family, socktype, proto, canonname, sockaddr = res
-                    self.address_deque.appendleft((sockaddr[0], sockaddr[1], True))
+                    self.address_deque.appendleft((sockaddr[0], sockaddr[1], host, True))
```

Comment:
I think using `hostname` is better than `host` here, even though they have the same value at this point, it is safer if the code is changed and they has different value in the future.